app/controllers/home_controller.rb

```diff
@@ -143,7 +143,7 @@ def account
       return
     end
 
-    @user = LinkedData::Client::Models::User.get(session[:user].id, include: 'all')
+    @user = LinkedData::Client::Models::User.get(session[:user].id, include: 'all', invalidate_cache: true)
```

Comment:
you can't invalidate the cache always, otherwise, why do we caching enabled. 
You need to invalidate only in the case where you need it?

Reply:
When you subscribe to an ontology you supposed to see it instantly added to your profile and when u refrech the page to see in the button 'unwatch' not 'watch'.
invalidate_cache: true, will invalidate cache only for this request? 
 'invalidate only in the case where you need it' for this endpoint the cache need to be always invalidate

Reply:
first `def account` is removed in https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/685/files   and this action `account` is called each time you go to your user page. Which will make it a lot slower each time

Reply:
and see maybe also https://github.com/ontoportal-lirmm/ontologies_api_ruby_client/blob/master/lib/ontologies_api_client/models/user.rb#L23

Reply:
Yes, I didn't see it a big deal since this call isn't slow or something.
the only solution to not use this is to invalidate the cache of the current user for the endpoing '/user..' in the moment of the update here
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/61744974/41a95f7f-c0e2-4660-8d26-b8b4bbedd201)
can I pass the param invalidate_cache to the update method here?

Reply:
> The only solution to not use this is to invalidate the cache of the current user for the endpoing '/user..' in the moment of the update here

That is why I pointed out this link
https://github.com/ontoportal-lirmm/ontologies_api_ruby_client/blob/master/lib/ontologies_api_client/models/user.rb#L23

this function is called each time you save/update a user

Reply:
in this case I will remove this invalidate cache param here in the UI, and the work should be done in the ruby client project